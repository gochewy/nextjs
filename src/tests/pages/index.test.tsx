import renderer from 'react-test-renderer';
import React from "react";
import {cleanup} from "@testing-library/react";
import Home from "../../../pages";

describe('Landing page', () => {
    afterEach(cleanup);


    let component;
    component = renderer.create(
            <Home />
    )
    it('should match snapshot', () => {
        expect(component.toJSON()).toMatchSnapshot();
    });
});