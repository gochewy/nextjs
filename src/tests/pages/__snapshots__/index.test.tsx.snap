// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should match snapshot 1`] = `
<div
  style={
    Object {
      "margin": "auto",
      "maxWidth": 600,
      "padding": "50px",
    }
  }
>
  <div>
    <p>
      Name: 
      joe
    </p>
    <p>
      Email: 
      hhhh
    </p>
    <p>
      ID: 
      fffff
    </p>
    <p>
      Token:
      <br />
      dfdfdfdf
    </p>
    <button
      name="Logout"
      onClick={[Function]}
    >
      Logout
    </button>
  </div>
</div>
`;
